app/app.js

```diff
@@ -1,15 +1,49 @@
 const express = require('express')
+const fs = require('fs/promises')
 
 const app = express();
 
 // const handleErrors = require('errorhandler.js')
 
-const {getTopics} = require('./controllers/topics.controller')
+const {getTopics} = require('./controllers/topics.controller');
+const { Console } = require('console');
+
+
+
+
+
 
 
 // app.use(handleErrors)
 
 
 app.get('/api/topics',getTopics)
 
+//reads the file
+//returns the endpoints.json file when you make this request
+//don't know if i have actually changed it or what 
+
+app.get('/api', (req,res) => {
+    fs.readFile('endpoints.json','utf-8',(err,data) => {
```

Comment:
Two things with this:
- it would be good to extract this function to another file and require it in, similar to how you do with controllers
- readFile is fine as a method for getting the content, but you can also just require the _endpoints_ file in and then send it 👍